### ISSUE TITLE
Extend Metrics API to accommodate street segment-level metrics / aggregates

### ISSUE THREAD

**Issue by jfh01** [openmobilityfoundation/mobility-data-specification#556]
### Is your feature request related to a problem? Please describe.

The Metrics API proposed in #485/#486/#487 offers a mechanism for exchange calculated / aggregated metrics within MDS. It supports metrics reporting across various dimensions, such as vehicle_type, MDS Geography, or provider. In its current form, MDS Geographies are not well suited for identifying individual street segments in a form that could be related to a linear referencing system. For some metrics (vehicle volume, average speed, parked vehicle density, etc.) street-segment level data could be extraordinarily useful.

This proposal is a placeholder for a conversation to extend Metrics to allow for aggregated metrics reporting at the street segment level, perhaps by creating an appropriate Dimension within the Metrics API.

**Comment by karcass** [comment]
Good question for the Curb WG

**Comment by schnuerle** [comment]
See notes from the [WG call](https://github.com/openmobilityfoundation/mobility-data-specification/wiki/Web-conference-notes,-2021.01.14-(Joint-Working-Group)#agenda) this week. 

- Sharada - streets segments identified with shared streets ID?
- Burden on providers?
- DC can do this if the GPS accuracy is good enough
- 2 things - disaggregated data with street segments and /trips, and in Metrics to aggregate to street segments (useful for providers offering to cities, not how it is now).
- Shared streets only ID used? City IDs. Question of analytic tools cities are using.
- Combine multiple provider data into Metrics
- Instead of geographies, talk in segments
- Use case - MPO wanting to make use of this for members
- OSM, Coord, CurbLR street segment systems, cities have their own segment ids in Open data. 
- Curb WG will decide how to define linear space, could be used here, needs to be consistent with rest of MDS

**Comment by quicklywilliam** [comment]
@schnuerle is this still under consideration for next release? It sounds like it might be blocked by the Curb WG making a decision about a linear referencing system?

**Comment by schnuerle** [comment]
@quicklywilliam It can be discussed especially if people have a good proposal for it. But it may change based on Curb efforts which can bring some clarity to a good technical solution.

**Comment by quicklywilliam** [comment]
@schnuerle we are interested in putting something together. Our thought is that it could be agnostic to the LRS – we'd focus on the aggregation method/format, and leave it open to be used with OSM, CurbLR, Coord etc.

**Comment by schnuerle** [comment]
Closing unless people chime in to say this is still needed, and have an idea on how it could be executed.
